source/ota_mqtt.c

```diff
@@ -628,7 +628,7 @@ static uint32_t buildStatusMessageReceiving( char * pMsgBuffer,
     received = numBlocks - pOTAFileCtx->blocksRemaining;
 
     /* Output a status update once in a while. */
```

Comment:
Can you update the comment to reflect the changes done?

Reply:
Comment updated in the #425.

Reply:
I see PR is closed. Is the plan to update this PR with the changes?

Reply:
I see PR is closed. Is the plan to update this PR with the changes?

Reply:
Sorry for late response, I created a new PR for this at #426.